src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -156,71 +101,226 @@ your domain. Be sure to include the escaping characters.
+
+<!--  Windows App -->
+
+## Check if your Windows (UWP) app is installed {: #check-windows }
+
+Your website can check if your Windows app (built using UWP) is installed.
+
+* Windows: Edge 85 or later
+
+### Tell your Windows app about your website
+
+You'll need to update your Windows app to define the relationship between your
+website and Windows application using [URI Handlers][win-uri-handlers]. This
+ensures that only your website can check if your Windows app is installed.
+
+Add the `Windows.appUriHandler` extension registration to your app's manifest
+file `Package.appxmanifest`. For example, if your website's address is
+`example.com` you would add the following entry in your app's manifest:
+
+```xml
+<Applications>
+  <Application ... >
+      ...
+      <Extensions>
+         <uap3:Extension Category="windows.appUriHandler">
+          <uap3:AppUriHandler>
+            <uap3:Host Name="example.com" />
+          </uap3:AppUriHandler>
+        </uap3:Extension>
+      </Extensions>
+  </Application>
+</Applications>
+```
 
-To define the relationship to an installed PWA, add a `related_applications`
-entry in the web app manifest, set `"platform": "webapp"` and provide
-the full path to the PWAs web app manifest in the `url` property.
+Then, create a JSON file (without the `.json` file extension) named
+`windows-app-web-link` and provide your app's package family name. Place
+that file either on your server root, or in the `/.well-known/` directory. You
+can find the package family name in the Packages section in the app manifest
+designer.
+
+```json
+[{
+  "packageFamilyName": "MyApp_9jmtgj1pbbz6e",
+  "paths": [ "*" ]
+ }]
+ ```
+
+See [Enable apps for websites using app URI handlers][win-uri-handlers] for
+complete details on setting up URI handlers.
+
+### Tell your website about your Windows app
+
+Next, tell your website about your Windows app by
+[adding a web app manifest](/add-manifest/) to your page. The manifest must
+include `related_applications` property, an array that provides the details
+about your app, including `platform` and `id`.
+
+* `platform` must be `windows`
+* `id` is your app's package family name
+
+```json
+{
+  "related_applications": [{
+    "platform": "windows",
+    "id": "MyApp_9jmtgj1pbbz6e",
+  }]
+}
+```
+
+You can find the package family name in the Packages section in the app
+manifest designer.
+
+### Check if your app is installed
+
+Finally, call [`navigator.getInstalledRelatedApps()`](#use) to check if your
+Windows app is installed.
+
+<!--  PWA - in scope -->
+
+## Check if your Progressive Web App is already installed (in scope) {: #check-pwa-in-scope }
+
+Your PWA can check to see if it is already installed. In this case, the page
+making the request must be on the same domain, and within the [scope][scope]
+of your PWA, as defined by the scope in the web app manifest.
+
+* Android: Chrome 84 or later
+* Windows: Edge 85 or later
```

Comment:
Checking for installed PWAs isn't supported on Windows yet. It's only on Android for now.

Reply:
Fixed - thanks!